__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const app = require('../app.js');
+const db = require("../db/connection.js");
+const testData = require('../db/data/test-data/index.js');
+const seed = require('../db/seeds/seed.js');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe("GET /api/healthcheck", () => {
+    test("returns 200 status code", () => {
+      return request(app).get("/api/healthcheck").expect(200);
+    });
+  });
+
+describe("GET /api/topics", () => {
+    test("return array of topic objects with properties slug and description", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+            const topics = response.body;
+            expect(topics.length).toBe(3)
+            topics.forEach((topic) => {
+                expect(topic.hasOwnProperty('slug')).toBe(true);
+                expect(topic.hasOwnProperty('description')).toBe(true);
+            })
+        })
+    })
+
+    test.only("returns 404 Not Found if path is not a route", () => {
+        return request(app)
+        .get('/api/topicssss')
```

Comment:
This is a good test, as an FYI this test for an invalid path is only needed once in your API, you don't need to test it again on future endpoints now that you have it